### PR TITLE
Update of Vagrant defaults

### PR DESCRIPTION
**Pull request by jirihnidek**
* Run deploy script with -t option to import test data to database.

### CODE REVIEW

File: vagrant/roles/candlepin-user/defaults/main.yml [hunk 1]
```diff
@@ -7,4 +7,4 @@ candlepin_user_home:  /home/vagrant
 candlepin_git_repo: https://github.com/candlepin/candlepin.git
 candlepin_checkout: /vagrant
 
-candlepin_deploy_args: "-g -a"
+candlepin_deploy_args: "-g -a -t"
```
Comment (barnabycourt) [inline vagrant/roles/candlepin-user/defaults/main.yml]: If someone doesn't want to deploy the test data, how would they do so?

Comment (jirihnidek) [reply vagrant/roles/candlepin-user/defaults/main.yml]: Good point. My change is overkill. I will only change `README.md` with instruction how to deploy with test data.

Comment (barnabycourt) [reply vagrant/roles/candlepin-user/defaults/main.yml]: That would be great.

**Comment by mstead** [comment]
> This does not appear to update the instructions in the vagrant readme. Is it really the best choice to always deploy the test data by default as opposed to the default being a clean install & letting someone override to set up test data if they want it.
> 
> @mstead What is your take on the default behavior for the CP vagrant environment?

Agree with @barnabycourt  on this.

File: vagrant/README.md [hunk 2]
```diff
@@ -54,6 +54,19 @@ to be stopped or you will have to edit the Vagrantfile and choose different port
 1. `buildr clean test=no package`
 1. `./server/bin/deploy`
 
+## Deploy with test data
+Candlepin Vagrant deployer deploy candlepin without any test data in database.
```
Comment (barnabycourt) [inline vagrant/README.md]: "The Candlepin Vagrant deployer will deploy.."

**Pull request merged**